### PR TITLE
Support for Sky130 OSU 12T hs, bug fixed on RO placement script

### PR DESCRIPTION
**Pull request by tuohangzeng**

### CODE REVIEW

**Review by msaligane — CHANGES_REQUESTED** [review]
@tuohangzeng You are pushing multiple files that are not needed (.libs/.db corners) and we need one merged gds instead of having the gds files for each instance (please check the hd library for reference).

**Comment by msaligane** [comment]
@tuohangzeng Please let me know when this is ready to for review/merge.

**Comment by tuohangzeng** [comment]
@msaligane The PR is ready for review. Thanks!

File: common/platforms/sky130osu12Ths/config.mk [hunk 1]
```diff
@@ -0,0 +1,111 @@
+# cell prefix
+export cell_prefix = sky130_osu_sc_12T_hs
```
Comment (msaligane) [inline common/platforms/sky130osu12Ths/config.mk:2]: can you use 12t instead of 12T (everywhere else). The capital is standing out ..

**Review by msaligane — CHANGES_REQUESTED** [review]
LGTM! Please add that small change and should be good to merge. Thanks @tuohangzeng

File: common/platforms/sky130osu12Ths/config.mk (hunk 1 above)
Comment (tuohangzeng) [reply common/platforms/sky130osu12Ths/config.mk:2]: Certainly!

Comment (msaligane) [reply common/platforms/sky130osu12Ths/config.mk:2]: I just realized that it comes like that from pdk.. please dismiss this change.

**Pull request merged**